wdn/templates_5.0/js-src/notice.babel.js

```diff
@@ -0,0 +1,331 @@
+let notices = document.querySelectorAll('[data-widget="notice"]');1111
+let fixedBottomExists = false; // flag for checking if a fixed bottom has been initialized on the page
+fixedBottomExists = document.querySelector('[id*="unl-widget-fixedBottom-"]') ? true : false; // if a fixedBottom notice has been previously added to the page set flag to true
+
+
+notices = [].slice.call(notices);
```

Comment:
I prefer working with arrays since it has more prototype methods than NodeLists (e.g. map, reduce, filter). NodeLists just got foreach too! but of course not all browsers support that.